modules/3d-tiles/src/tile-3d-subtree-loader.ts

```diff
@@ -1,12 +1,12 @@
-import type {LoaderWithParser} from '@loaders.gl/loader-utils';
+import type {LoaderOptions, LoaderWithParser} from '@loaders.gl/loader-utils';
+import type {Subtree} from './types';
 import parse3DTilesSubtree from './lib/parsers/helpers/parse-3d-tile-subtree';
 import {VERSION} from './lib/utils/version';
 
 /**
  * Loader for 3D Tiles Subtree
- *
  */
-export const Tile3DSubtreeLoader: LoaderWithParser = {
+export const Tile3DSubtreeLoader: LoaderWithParser<Subtree, never, LoaderOptions> = {
```

Comment:
- Return types of the loaders should be exported from the module on the top level. 
- We should probably rename this to `Tiles3DSubtree` if we make it public.
-  We should define and export a `Tiles3DLoaderOptions` even if we don't have any unique options now. That lets us add additional options later in a non-breaking way.